### PR TITLE
[3792] Changed distance around "Have a discount code?" as per design (recreated)

### PR DESCRIPTION
**Pull request by S-Arturs**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3792

**Problem:**
* "Have a discount code?" is wrong distance from elements around

**In this PR:**
* Changed margins and paddings around "Have a discount code?" and nearby elements

### CODE REVIEW

File: packages/scandipwa/i18n/fa_IR.json [hunk 1]
```diff
@@ -1,307 +1,121 @@
 {
-    "Something went wrong!": "با عرض پوزش خطایی رخ داده است!",
```
Comment (tatiana-scandi) [inline packages/scandipwa/i18n/fa_IR.json:2]: Why did you remove all the translated strings and regenerated translation file? (I suppose it was by accident).

**Review by tatiana-scandi — CHANGES_REQUESTED** [review]
CR failed, needs updates

File: packages/scandipwa/i18n/fa_IR.json (hunk 1 above)
Comment (S-Arturs) [reply packages/scandipwa/i18n/fa_IR.json:2]: Aleksandra asked me to, it was impossible to test for some reason

**Review by tatiana-scandi — APPROVED** [review]
